doc/sphinx/conf.py.in

```diff
@@ -168,10 +168,12 @@ man_pages = [
     ("docs/fpga_tools/userclk/userclk", 'userclk', u'Set AFU high and low clock frequency', [author], 8),
     ("docs/fpga_tools/pci_device/pci_device", 'pci_device', u'Common operations for PCIe devices', [author], 8),
     ("docs/fpga_tools/opae.io/opae.io", 'opae.io', u'User space access to PCIe devices', [author], 8),
-    ("docs/fpga_tools/host_exerciser/host_exerciser.md", 'host_exerciser', u'Host Exerciser lpbk and memory', [author], 8),
-    ("docs/fpga_tools/hssi_ethernet/hssistats.md", 'hssistats', u'High Speed Serial Interface ethernet statistics', [author], 8),
-    ("docs/fpga_tools/hssi_ethernet/hssimac.md", 'hssimac', u'High Speed Serial Interface ethernet max', [author], 8),
-    ("docs/fpga_tools/hssi_ethernet/hssiloopback.md", 'hssiloopback', u'High Speed Serial Interface ethernet loopback', [author], 8),
+    ("docs/fpga_tools/host_exerciser/host_exerciser", 'host_exerciser', u'Host Exerciser lpbk and memory', [author], 8),
+    ("docs/fpga_tools/hssi_ethernet/hssistats", 'hssistats', u'High Speed Serial Interface ethernet statistics', [author], 8),
+    ("docs/fpga_tools/hssi_ethernet/hssimac", 'hssimac', u'High Speed Serial Interface ethernet max', [author], 8),
+    ("docs/fpga_tools/hssi_ethernet/hssiloopback", 'hssiloopback', u'High Speed Serial Interface ethernet loopback', [author], 8),
+    ("docs/fpga_tools/opaeuio/opaeuio", 'opaeuio', u'Binding and Unbinding of a DFL device', [author], 8),
```

Comment:
This tool (opaeuio script) no longer exists.  Please remove it.

Reply:
removed opaeuio